File: C_C++_Programs/Basic_Programs/Reverse_Number.c
#include<stdio.h>

int main(){

    int num;

    printf("Enter  Number : ");
    scanf("%d",&num);

    int reverse_num=0;

    while (num!=0)
    {
        int last_digit=num%10;
        reverse_num=(reverse_num*10)+last_digit;
        num/=10;
    }

    printf("\n Reverse Digit : %d",reverse_num);
    


}

File: Simple_Program/Day_1/Reverse_Number.java
package Day_1;
import java.util.Scanner;

public class Reverse_Number {
    public static void main(String[] args) {
        
       Scanner sc=new Scanner(System.in);

       System.out.println("Enter Number : ");
       int num=sc.nextInt();

       int reverse_num=0;

       while (num!=0) {
           int last_digit=num%10;
           reverse_num=(reverse_num*10)+last_digit;
           num/=10;
       }

       System.out.println("Reverse Number : "+reverse_num);

         

    }
}
